### DISCUSSION TITLE
needs better suport for

### DISCUSSION THREAD

**Discussion by DeviateBacon3** [ThioJoe/YT-Spammer-Purge#880] [General]
python 3.10 becoes my OS has python 3.10 but it say you may run this script again inside your installaion to update.

**Comment by Firecul** [comment]
Are you referring to https://github.com/ThioJoe/YT-Spammer-Purge#updating ?

![image](https://user-images.githubusercontent.com/34950371/177454255-c1fdb220-cc3a-47c8-9560-ab7712b829f4.png)

---

If you aren't I'm not sure what you mean. I'm also running Python 3.10 with no issues, this script can't (or rather shouldn't be able to) update Python on your computer.

![image](https://user-images.githubusercontent.com/34950371/177454191-011ee616-7f20-42a6-b75d-65034a45fcc9.png)

Again, if I'm misunderstanding what you mean I'm happy to listen to more comments.
